hera_mc/geo_handling.py

```diff
@@ -253,7 +259,7 @@ def find_station_of_antenna(self, antenna, query_date):
             raise ValueError('More than one active connection between station and antenna')
         return antenna_connected.upstream_part
 
-    def get_location(self, to_find, query_date, show_location=False, verbosity='m'):
+    def get_location(self, to_find, query_date, station_types=None, show_location=False, verbosity='m'):
```

Comment:
`station_types` should be added to this function's list of parameters in the docstring.